flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/BoundedBlockingSubpartitionReader.java

```diff
@@ -44,7 +44,7 @@
 	/** The reader/decoder to the memory mapped region with the data we currently read from.
 	 * Null once the reader empty or disposed.*/
 	@Nullable
-	private BufferSlicer memory;
+	private BoundedData.Reader data;
```

Comment:
Maybe “dataReader” is better than “data” to distinguish from instance of  BoundedData.